util/test/prediff-for-knl_unexpected_cache_size

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+# This script is for the use with 'start_test' as the system-wide prediff
+# when running KNL tests on purie.
+# To use (assuming bash):
+#
+#  CHPL_SYSTEM_PREDIFF=$CHPL_HOME/util/test/prediff-for-knl_unexpected_cache_size \
+#    start_test ...
+
+sed -e '/Unexpected KNL MCDRAM cache size/d' $outfile >$outfile.tmp
+mv -f $outfile.tmp $outfile
```

Comment:
There is one line missing to make this work.
```sh
outfile=$2
```
This should appear before the `sed` line.